src-docs/webpack.config.js

```diff
@@ -40,149 +40,133 @@ function employCache(loaders) {
-        loader: 'file-loader',
-      },
-      {
-        test: /\.(png|jp(e*)g|svg|gif)$/,
-        loader: 'url-loader',
-        options: {
-          limit: 8000, // Convert images < 8kb to base64 strings
-          name: 'images/[hash]-[name].[ext]',
         },
-      },
+      ],
+    },
+
+    plugins: [
+      new HtmlWebpackPlugin({
+        template: 'index.html',
+        inject: 'body',
+        cache: true,
+        showErrors: true,
+      }),
+
+      new CircularDependencyPlugin({
+        exclude: /node_modules/,
+        failOnError: true,
+      }),
+
+      new NodePolyfillPlugin(),
+      // run TypeScript during webpack build
+      // new ForkTsCheckerWebpackPlugin({
+      //   typescript: { configFile: path.resolve(__dirname, '..', 'tsconfig.json') },
+      //   async: false, // makes errors more visible, but potentially less performant
+      // }),
     ],
-  },
-
-  plugins: [
-    new HtmlWebpackPlugin({
-      template: 'index.html',
-      inject: 'body',
-      cache: true,
-      showErrors: true,
-    }),
-
-    new CircularDependencyPlugin({
-      exclude: /node_modules/,
-      failOnError: true,
-    }),
-
-    // run TypeScript during webpack build
-    // new ForkTsCheckerWebpackPlugin({
-    //   typescript: { configFile: path.resolve(__dirname, '..', 'tsconfig.json') },
-    //   async: false, // makes errors more visible, but potentially less performant
-    // }),
-  ],
-
-  devServer: isDevelopment
-    ? {
-        contentBase: 'src-docs/build',
-        host: '0.0.0.0',
-        allowedHosts: ['*'],
-        port: getPortSync({
-          port: getPort.makeRange(8030, 8130),
+
+    devServer: isDevelopment
+      ? {
+          static: {
+            directory: path.resolve(__dirname, 'build'),
+            // prevent file watching while running on CI
+            // /app/ represents the entire docker environment
+            watch: isPuppeteer ? { ignored: '**/*' } : undefined,
+          },
           host: '0.0.0.0',
-        }),
-        disableHostCheck: true,
-        historyApiFallback: true,
-        // prevent file watching while running on CI
-        // /app/ represents the entire docker environment
-        watchOptions: isPuppeteer
-          ? {
-              ignored: '**/*',
-            }
-          : undefined,
-      }
-    : undefined,
-  node: {
-    fs: 'empty',
-  },
+          port,
+          allowedHosts: 'all',
+          historyApiFallback: true,
+        }
+      : undefined,
+  };
 };
 
-// Inspired by `get-port-sync`, but propogates options
-function getPortSync(options) {
-  let isDone = false;
-  let freeport = null;
-  let error = null;
-
-  getPort(options)
-    .then((port) => {
-      isDone = true;
-      freeport = port;
-    })
-    .catch((err) => {
-      isDone = true;
-      error = err;
-    });
-
-  // wait until we're done'
-  deasync.loopWhile(() => !isDone);
-
-  if (error) {
-    throw error;
-  } else {
-    return freeport;
-  }
-}
```

Comment:
`getPort` was not running for reasons I do not know, so I made `webpack.config.js` as a function and removed `deasync`